website/landingpage/public/blog/2020-06-18-welcome-to-luigi-blog.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
The blog pages, for now I think should stay in **luigi/blog** instead of **luigi/website/landingpage/public/blog/**

Reply:
Correct, put blog files only in root of luigi /blog

Reply:
Thanks! I can't find the *luigi/blog* folder, should I create it?